.circleci/config.yml

```diff
@@ -91,6 +91,9 @@ jobs:
       - run:
           name: run linter
           command: git ls-files | grep '\.l\?hs$' | xargs stack exec -- hlint -X QuasiQuotes -X NoPatternSynonyms "$@"
+      - run:
+          name: run styler
+          command: git ls-files | grep '\.l\?hs$' | xargs stack exec -- stylish-haskell -i && git diff-index --quiet HEAD --
```

Comment:
Maybe use `--exit-code` instead of  `--quiet`. This is a new workflow for me, if there's an error I'd like to know in which file there's a discrepancy.

Reply:
:100: